mlocal/frags/build_network.mk

```diff
@@ -57,12 +58,12 @@ cni_config_INSTALL := $(DESTDIR)$(SYSCONFDIR)/singularity/network
 .PHONY: cniplugins
 cniplugins:
 	$(V)install -d $(cni_builddir)
-	$(V)for p in $(cni_plugins_GOPATH); do \
+	$(V)for p in $(cni_plugins); do \
 		name=`basename $$p`; \
 		cniplugin=$(cni_builddir)/$$name; \
 		if [ ! -f $$cniplugin ]; then \
 			echo " CNI PLUGIN" $$name; \
-		go build $(GO_BUILDMODE) -tags "$(GO_TAGS)" $(GO_LDFLAGS) $(GO_GCFLAGS) $(GO_ASMFLAGS) \
+		go build $(GO_MODFLAGS) $(GO_BUILDMODE) -tags "$(GO_TAGS)" $(GO_LDFLAGS) $(GO_GCFLAGS) $(GO_ASMFLAGS) \
```

Comment:
```suggestion
		GOPATH=$(SOURCEDIR)/vendor go build $(GO_MODFLAGS) $(GO_BUILDMODE) -tags "$(GO_TAGS)" $(GO_LDFLAGS) $(GO_GCFLAGS) $(GO_ASMFLAGS) \
```

Reply:
Thanks! I'm going to guess you ran into the same issue I did, where sometimes the CNI plugins couldn't be found.

Experimenting around, setting GO111MODULE=on explicitly seems to take care of this in all cases.